assets/scss/components/_navbar.scss

```diff
@@ -73,6 +73,20 @@ $zindex--aboveFloatingContent: map-get($zindex-map, "floating-content") + 1;
 	}
 }
 
+.navItem--counterBadge {
+	$_newDotSize: 14px;
+	background-color: rgb(246, 88, 88);
```

Comment:
Do we typically use rgb values or hex codes for colors?

Reply:
In build-meetup I see hex code for this color: peach: '#F65858', will refactor